guides/common/modules/proc_migrating-project-or-smartproxy-using-clone.adoc

```diff
@@ -0,0 +1,24 @@
+[id="migrating-{project-context}-or-proxy-using-clone_{context}"]
+= Migrating {Project} or {SmartProxy} to {EL-abbr}{nbsp}9 by using cloning
+
+You can clone your existing {ProjectServer} from {EL}{nbsp}8 to a freshly installed {EL}{nbsp}9 system.
+ifdef::satellite[]
+Create a backup of the existing {ProjectServer}, which you then clone on the new {EL}{nbsp}9 system.
+[NOTE]
+====
+You can use cloning for {ProjectServer} backups but not for {SmartProxyServer} backups.
+====
+endif::[]
+
+.Procedure
+. Perform a full backup of your {ProjectServer}.
+This is the source {EL}{nbsp}8 server that you are migrating.
+For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin#Performing_a_Full_Backup_admin[Performing a full backup of {ProjectServer}] in _{AdministeringDocTitle}_.
```

Comment:
Wrong link? It looks like it has two anchors. Perhaps a copy-pasting mistake?

Reply:
Yes! the anchors were the issue! Thanks